search_service/__init__.py

```diff
@@ -3,14 +3,12 @@
 INDEX_NAME = "database"
 DOC_TYPE = "json"
 
-ELASTIC_HOST = os.environ['ELASTIC_HOST']
-ELASTIC_PORT = os.environ['ELASTIC_PORT']
+HOST = "0.0.0.0"
```

Comment:
`HOST` is too generic. Please specify it a bit more. Which host? Why do I need this variable?

Reply:
We need 'HOST' to seed the database and to use GraphQl.